source/06_monitoring_features/external_commands.rst

```diff
@@ -0,0 +1,62 @@
+.. _monitoring_features/external_commands:
+
+=================
+External Commands
+=================
+
+
+Introduction
+------------
+
+Alignak can process commands from external applications to alter various aspects of its monitoring functions based on the commands it receives.
+
+
+Enabling External Commands
+--------------------------
+
+.. image:: /_static/images///official/images/externalcommands.png
+   :scale: 90 %
+
+
+In order to have Alignak process external commands, make sure you do the following:
+
+    * Enable external command checking with the ``check_external_commands`` parameter in the monitoring configuration file.
+    * Install an external commands capable module near the Alignak receiver daemon.
+
+An external commands capable module implements a solution to provide the external commands to Alignak.
+
+The :ref:`external commands named pipe module <modules/named_pipe>` allows an external application residing on the same host as Alignak to simply write the external commands directly to a named pipe file as outlined above. However, applications on remote hosts can't do this so easily.
+
+To allow remote hosts to send passive check results to the Alignak, the :ref:`Web services module <modules/web_services>` exposes a Web service
```

Comment:
Yes, the Web service module allows to send external commands to Alignak: https://github.com/Alignak-monitoring-contrib/alignak-module-ws#short-description

NSCA only allows external commands for passive checks.